### PR TITLE
docs(720-to-721): Add Model-API HTTL Documentation

### PR DESCRIPTION
**Pull request by psavidis**
Related-to: https://github.com/camunda/camunda-bpm-platform/issues/3764

### CODE REVIEW

File: content/update/minor/720-to-721/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+This document guides you through the update from Camunda Platform `7.20.x` to `7.21.0` and covers the following use cases:
+
+1. For Developers: [Add Default History Time To Live to BPMN Fluent API](#add-default-history-time-to-live-to-bpmn-fluent-api)
+
+This guide covers mandatory migration steps and optional considerations for the initial configuration of new functionality included in Camunda Platform 7.21.
+
+# Add Default History Time To Live to BPMN Fluent API
+
+Starting with this release, the `BPMN Fluent API Builder` assigns by default a `historyTimeToLive` of **6 months** to processes. You can change this default or pass `null` using the `#setCamundaHistoryTimeToLiveString` API.
```
Comment (danielkelemen) [inline content/update/minor/720-to-721/_index.md:24]: 🔧 I wouldn't code format everything and maybe clarify what `null` does.
- BPMN Fluent API Builder could be a link [to the corresponding docs](https://docs.camunda.org/manual/7.20/user-guide/model-api/bpmn-model-api/fluent-builder-api/).
- `historyTimeToLive`: the actual attribute is `camundaHistoryTimeToLive`.

```suggestion
Starting with this release, the [BPMN Fluent API Builder](https://docs.camunda.org/manual/7.20/user-guide/model-api/bpmn-model-api/fluent-builder-api/) assigns by default a history time to live of **6 months** to processes. You can change this default or pass `null` to remove the attribute using the `#setCamundaHistoryTimeToLive` or `#setCamundaHistoryTimeToLiveString` API.
```

(The link needs adjustment)

**Review by danielkelemen — APPROVED** [review]
Looks generally good, I added some suggestions to make it clearer and match the formatting.

**Review by danielkelemen — CHANGES_REQUESTED** [review]
❌ Since you are creating the `720-to-721` folder, you also need to copy the other pages and add the default migration use-cases like database update etc. from the previous guide.

File: content/update/minor/720-to-721/_index.md (hunk 1 above)
Comment (psavidis) [reply content/update/minor/720-to-721/_index.md]: I've used the suggestion with a relative URL

**Comment by psavidis** [comment]
> ❌ Since you are creating the `720-to-721` folder, you also need to copy the other pages and add the default migration use-cases like database update etc. from the previous guide.

The rest of AS pages are copied and the version is changed to 7.21. Just a note for the review: the URLs pointing to the 7.21 download artifacts are not yet working.

File: content/update/minor/720-to-721/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    name: "7.20 to 7.21"
+    identifier: "migration-guide-721"
+    parent: "migration-guide-minor"
+    pre: "Update from `7.20.x` to `7.21.0`."
+
+---
+
+This document guides you through the update from Camunda Platform `7.20.x` to `7.21.0` and covers the following use cases:
+
+1. For Developers: [Add Default History Time To Live to BPMN Fluent API](#add-default-history-time-to-live-to-bpmn-fluent-api)
```
Comment (danielkelemen) [inline content/update/minor/720-to-721/_index.md:18]: ❌ Please also add the [first two bullet points](https://github.com/camunda/camunda-docs-manual/blob/master/content/update/minor/719-to-720/_index.md?plain=1#L18) from the previous guide.

**Review by danielkelemen — CHANGES_REQUESTED** [review]

File: content/update/minor/720-to-721/_index.md (hunk 2 above)
Comment (psavidis) [reply content/update/minor/720-to-721/_index.md:20]: Added

File: content/update/minor/720-to-721/_index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+This section is applicable if you installed the
+[Full Distribution]({{< ref "/introduction/downloading-camunda.md#full-distribution" >}})
+with a **shared process engine**.
+
+The following steps are required:
+
+1. Update the Camunda libraries and applications inside the application server.
+2. Migrate custom process applications.
+
+Before starting, ensure you have downloaded the Camunda Platform 7.20 distribution for the application server you use. This contains the SQL scripts and libraries required for the update. This guide assumes you have unpacked the distribution to a path named `$DISTRIBUTION_PATH`.
```
Comment (danielkelemen) [inline content/update/minor/720-to-721/_index.md:40]: 🔧

```suggestion
Before starting, ensure you have downloaded the Camunda Platform 7.21 distribution for the application server you use. This contains the SQL scripts and libraries required for the update. This guide assumes you have unpacked the distribution to a path named `$DISTRIBUTION_PATH`.
```

**Review by danielkelemen — APPROVED** [review]
Looks good 👍
One last version number needs adjusted.

**Pull request merged**